### ISSUE TITLE
The scan works only  when the background is dark

### ISSUE THREAD

**Issue by hassan379** [RatScanner/RatScanner#660]
**Describe the bug**
The scan works only  when the background is dark, i have to turn the entire player to a place has dark background to scan correctly
if the background is bright its not working, but its working in stash

**To Reproduce**
just try to loot an item and scan it when the background is bright 

**Expected behavior**
it should scan the items in all situations

**Comment by Blightbuster** [comment]
Yeah unfortunately this is how it currently is :( I've tried a few time to make a better algorithm for the inventory grid separation but none really hit the mark of working well enough for bright backgrounds while still detecting all normal (darker) grids.
